src/common/harvest/harvest.js

```diff
@@ -115,6 +117,8 @@ export class Harvest extends SharedContext {
       } else {
         body = stringify(body)
       }
+      /** Warn --once-- if the agent tries to send large payloads */
+      if (body.length > 750000 && warnings++ < 1) warn('The Browser Agent is attempting to send a very large payload. This is usually tied to large amounts of custom attributes. Please check your configurations.')
```

Comment:
There also should be no limit on warnings for visibility

Reply:
I hesitate because this check happens every harvest and doesn't provide new info each time, so it's likely to simply spam the customer's customer's console with 10-20 messages (or more) per page load with the same information

Reply:
What about once per endpoint then? (make warnings a set)

Reply:
sounds good, will do that